docs/workload/wordpress.md

```diff
@@ -0,0 +1,79 @@
+# Deploy WordPress with NFS Persistent Volumes
+This document explains how to deploy a WordPress site using OpenEBS NFS Volume. Since OpenEBS NFS volume supports RWX(ReadWriteMay) storage mode, WordPress deployment using OpenEBS NFS volumes is highly scalable.
+
+## Prerequisites
+Kubernetes cluster with OpenEBS NFS Provisioner installed. Refer [QuickStart guide on How to install OpenEBS NFS Provisioner](https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/docs/intro.md#quickstart)
+
+## Deploying WordPress
+We will use the Helm package to install WordPress in our kubernetes cluster. If you don't have Helm installed, follow the [Installing Helm](https://helm.sh/docs/intro/install/) guide for installation.
+
+### Adding Help repo for WordPress
+Use the below command to add helm repo for WordPress.
+
+```
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+Once repo has been added successfully, update helm repo using the following command:
+
+```
+helm repo update
+```
+
+### Installing WordPress
+Once the helm repo is added, you can install WordPress using `helm install` as mentioned below. In this command, we are using Storageclass `openebs-rwx` created using [QuickStart guide](https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/docs/intro.md#quickstart). If you have created a different OpenEBS NFS Storageclass then you need to update the value of `--set persistence.storageClass`.
+You can also configure [the other parameters through the `--set` argument](https://github.com/bitnami/charts/tree/master/bitnami/wordpress#parameters)
+
+```
+helm install my-release -n wordpress --create-namespace \
+       --set wordpressUsername=admin \
+       --set wordpressPassword=password \
+       --set mariadb.auth.rootPassword=secretpassword \
+       --set persistence.storageClass=openebs-rwx \
+       --set persistence.accessModes={ReadWriteMany} \
+       --set volumePermissions.enabled=true \
+       --set autoscaling.enabled=true \
+       --set autoscaling.minReplicas=1 \
+       --set autoscaling.maxReplicas=6 \
```

Comment:
nit: As per scale command below it can be 1
```suggestion
       --set autoscaling.maxReplicas=1 \
```

Reply:
Better to keep maxReplicas's as `2` here and then `3` in the scaled command below.

Reply:
ok

Reply:
removed scale command as autoscaling is enabled in helm install command